### PR TITLE
Add cents converter and dollar sign everywhere

### PR DESCRIPTION
**Pull request by carolynyoo**
screen shot 2015-04-28 at 9 53 10 pm

### CODE REVIEW

File: browser/js/films/films.html [hunk 1]
```diff
@@ -17,8 +17,10 @@
       <img ng-src="{{film.photo}}" />
       <p>{{film.title}}</p>
     </a>
-    <button type="button" class="btn btn-default">{{film.price}}</button>
+   <p><span ng-repeat="category in film.categories">
+    {{category.name}} {{$last ? '' : ($index==film.categories.length-2) ? ' &amp; ' : ','}}
```
Comment (itsaliu90) [inline browser/js/films/films.html]: Very cool, a little hard on the readability side, but certainly condensed.

**Comment by itsaliu90** [comment]
:hamburger:

File: browser/js/films/films.html (hunk 1 above)
Comment (carolynyoo) [inline browser/js/films/films.html]: ya open to changes. we can go back to the button view!

**Comment by carolynyoo** [comment]
revamp-film

After feedback, removed category from this page (they can just filter or see it on the product view). Also made responsive and tweaked some sizing.

**Comment by darksim735** [comment]
:hamburger:

**Comment by gskerry** [comment]
Have two duplicate cents converters? Can we consolidate? 
Otherwise, hamburgers.

**Pull request merged**